login-fire.html

```diff
@@ -84,25 +84,23 @@
     }
 
     .emailPWError {
-      color: #dd2c00;
-      direction: ltr;
+      color: var(--error-color);
+      text-align: center;
       font-size: 16px;
       font-weight: 500;
       line-height: 24px;
       margin: 0;
       padding: 0;
-      text-align: left;
     }
 
     .emailPWMessage {
-      color: #303030;
-      direction: ltr;
+      color: var(--primary-color);
```

Comment:
Hey, if I rename it, it will not have that eye catchy blue color. Or you just want me to define it and set to `--primary-color` ?